packages/keystone/lib/List/index.js

```diff
@@ -1094,6 +1094,24 @@ module.exports = class List {
       originalInput,
       actions: mapKeys(this.hooksActions, hook => hook(context)),
     };
+
+    // Check for isRequired
+    const fieldValidationErrors = this.fields
+      .filter(
+        f =>
+          f.isRequired &&
+          !f.isRelationship &&
```

Comment:
When is an `isRequired` relationship validated?

Reply:
Never :-) Need to create a separate issue for that, because as you noted above, we attempt to resolve relationships before the validation step.

Reply:
Got it 👍